doc/classes/Viewport.xml

```diff
@@ -116,6 +116,13 @@
 				Returns the visible rectangle in global screen coordinates.
 			</description>
 		</method>
+		<method name="gui_find_control">
+			<return type="Control" />
+			<param index="0" name="point" type="Vector2" />
+			<description>
+				Returns the top-level [Control] at the given point within this viewport. If no [Control] is at the point, returns null.
```

Comment:
This needs some more detail IMO, what does it mean "at the given point"? Does it mean:
* The point is within its bounds?
* Does it have a visible pixel in that point? (i.e. `_has_point` returns true)
* Or something else?

Details on this would be required to know how to use this properly

Reply:
Thanks for your help on fixing this description. I committed your suggested changes down below.